### ISSUE TITLE
Suggestion: Automatically create draft replies

### ISSUE THREAD

**Issue by zackkatz** [code-debug228/HostetskiGPT#18]
Unless I'm missing something, the ChatGPT reply generation requires interaction to generate and then copy/pasting into a reply.

It'd be great if there were a setting to enable auto-generating draft replies for each incoming ticket.

**Comment by code-debug228** [comment]
Yes, I think it's a good idea, I'll implement it in the next updates.
